docs/getting-started/components.mdx

```diff
@@ -0,0 +1,150 @@
+# Add Component
+
+To understand better, let's review a few tips before you can add your first component.
+
+## Components
+
+This documentation refers to widgets as components since design tools like Figma use this terminology to describe their reusable UI elements.
```

Comment:
Should we explain this here if it's part of the Terminology?

Reply:
Well, my concern was people may skip termonology. but we can write here to reference to component on that page instead. I agree.